src/seedu/addressbook/data/person/Address.java

```diff
@@ -55,4 +70,73 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+}
+
+class Block{
+	
+	private String blockNo;
+	
+	// Default Constructor
+	public Block(){
```

Comment:
- the comment adds little value. Make it more meaningful. May be explain why you gave a default value.